minecraft_servers/gameup/manifest.json

```diff
@@ -0,0 +1,87 @@
+{
+  "server_name": "gameup",
+  "nice_name": "§8§l[§c§lGame§f§lUP§8§l]§8",
```

Comment:
Just a normal name, without any color codes

Reply:
May i ask why, is that against rules or something? because minecraft fully supports that even older versions, and it give better looking to the server,
![image](https://github.com/user-attachments/assets/21541d8c-011a-432b-8ee7-342feb1f36b8)

Reply:
The name will be displayed on https://laby.net